packages/cheqd-sdk/src/anoncreds/utils/transform.ts

```diff
@@ -0,0 +1,255 @@
+// To parse this data:
```

Comment:
I have little understanding of what's going on in this file. Is this needed because the resource is returned in a different structure by the cheqd sdk? When we did the integration a while back I didn't seem to need these?

Or is this for validation of the objects? in that case we have `class-transformer` and `class-validator` which would be a bit more consinstent with the rest of the framework and hides a lot of the complexity.

Reply:
Yes this is just for json schema validation, great will have a look at them